demos/smartlab_demo/python/smartlab_demo.py

```diff
@@ -0,0 +1,147 @@
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import time
+from collections import deque
+from argparse import ArgumentParser, SUPPRESS
+from object_detection.detector import Detector
+from segmentor import SegmentorMstcn
+from evaluator import Evaluator
+from display import Display
+from openvino.runtime import Core
+
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                    help='Show this help message and exit.')
+    args.add_argument("-d", "--device", type=str, default='CPU', required=False,
+                        help="Optional. Specify the target to infer on CPU or GPU.")
+    args.add_argument('-tv', '--topview', required=True,
+                    help='Required. Topview stream to be processed. The input must be a single image, '
+                    'a folder of images, video file or camera id.')
+    args.add_argument('-fv', '--frontview', required=True,
+                    help='Required. FrontView to be processed. The input must be a single image, '
+                    'a folder of images, video file or camera id.')
+    args.add_argument('-m_ta', '--m_topall', help='Required. Path to topview all class model.', required=True, type=str)
+    args.add_argument('-m_tm', '--m_topmove', help='Required. Path to topview moving class model.', required=True, type=str)
+    args.add_argument('-m_fa', '--m_frontall', help='Required. Path to frontview all class model.', required=True, type=str)
+    args.add_argument('-m_fm', '--m_frontmove', help='Required. Path to frontview moving class model.', required=True, type=str)
+    args.add_argument('--mode', default='mstcn', help='Optional. action recognition mode: multiview or mstcn', type=str)
+    args.add_argument('-m_en', '--m_encoder', help='Required. Path to encoder model.', required=True, type=str)
+    args.add_argument('-m_de', '--m_decoder', help='Required. Path to decoder model.', required=True, type=str)
+
+    return parser
+
+def main():
+    args = build_argparser().parse_args()
+
+    frame_counter = 0 # Frame index counter
+    buffer1 = deque(maxlen=1000)  # Array buffer
+    buffer2 = deque(maxlen=1000)
+    ie = Core()
+
+    ''' Object Detection Variables'''
+    detector = Detector(
+            ie,
+            args.device,
+            [args.m_topall, args.m_topmove],
+            [args.m_frontall, args.m_frontmove])
+
+    '''Video Segmentation Variables'''
+    if(args.mode == "mstcn"):
```

Comment:
The `args.mode` should be removed, because it does not influent the choice of segmentor at all now